.buildkite/supported_plugins_test_pipeline.yml

```diff
@@ -0,0 +1,35 @@
+agents:
+  image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-logstash-ci:0.2"
```

Comment:
I think depends which is the intent of the pipeline. If it needs stable versions to avoid boundary error, then maybe it's best to pin the version. If, like this, the pipeline is intended to provide feedback on possible incompatibilities as soon as possible, then it's better to avoid pin. I'll un-pin

Reply:
Fixed by 5b9ff3b331f52c1622e3d1d64aeccfb9ddae7ea5